desktop-exporter/trace_store_test.go

```diff
@@ -106,7 +106,7 @@ func TestGetRecentTraceIDs(t *testing.T) {
 		store.Add(ctx, span)
 	}
 
-	recentTraceIDs := store.GetRecentTraceIDs(numRecentTraceIDs)
+	recentTraceIDs := store.getRecentTraceIDs(numRecentTraceIDs)
```

Comment:
If you are going to make `getRecentTraceIDs` a non-public interface, I would update this test to only use `store.GetRecentTraces()`